src/providers/editVideoSheet/hooks.tsx

```diff
@@ -70,7 +70,7 @@ export const useEditVideoSheetTabData = (tab?: EditVideoSheetTab) => {
     title: tab.isDraft ? draft?.title ?? '' : video?.title ?? '',
     description: (tab.isDraft ? draft?.description : video?.description) ?? '',
     category: (tab.isDraft ? draft?.category : video?.category?.id) ?? null,
-    licenseCode: (tab.isDraft ? draft?.licenseCode : video?.license?.code) ?? 1001, // PDM as a default license
+    licenseCode: (tab.isDraft ? draft?.licenseCode : video?.license?.code) ?? 1002,
```

Comment:
Could we maybe stick that into some constant like `DEFAULT_LICENSE_ID` to avoid having this repeated?